### ISSUE TITLE
Add basic auth

### ISSUE THREAD

**Issue by paullarionov** [gvalkov/tailon-legacy#16]
Hello Georgi,
Thanks for tailon, it is really cool!! Is it possible to add HTTP basic auth directly for tailon? 
Thanks.

**Comment by gvalkov** [comment]
Hello @paullarionov,

Sure, this makes sense. I'm in the process of writing a generic basic & digest auth module for tornado (modeled after bits of twisted's auth framework). Can't really tell when I'll be done with all this. Thanks for the patience.

**Comment by hugsbrugs** [comment]
Hi and thanks for this great work !
I'm also interested in having a basic auth !

I'm offering a workaround waiting for this : 
Create a domain or subdomain like tailon.mydomain.com
Create an apache virtual host

```
<VirtualHost *:80>
  ProxyPreserveHost On
  ProxyRequests Off
  ServerName tailon.mydomain.com
  ProxyPass / http://localhost:8080/
  ProxyPassReverse / http://localhost:8080/
  <Proxy *>
        Order deny,allow
        Allow from all
        AuthType Basic
        AuthName "Dev Server"
        AuthUserFile "/var/www/tailon/.htpasswd"
        Require valid-user
  </Proxy>
</VirtualHost>
```

Create according folder and file : /var/www/tailon/.htpasswd

Start tailon on localhost:8080

This way you can access your log files securely !

**Comment by dgvigil** [comment]
I would love to see this feature added as well.  Some logs (ie messages/syslog) are only viewable by root  for a reason and putting a simple auth would ensure that the logs aren't viewable by anyone crawling the site/server.

**Comment by wasimbaig** [comment]
Any update on basic auth, thanks for the tool

**Comment by gvalkov** [comment]
Well, it took me long enough, but it's finally done: https://github.com/gvalkov/tailon/commit/3ec5a52416fdb55d8f15d01c1bbe23f4fa17eef3

The feature is available in tailon 1.2.0, which was just pushed to PyPi.
Feedback and PRs are welcome.
